@navikt/core/react/src/modal/Modal.tsx

```diff
@@ -79,111 +94,153 @@ type ModalLifecycle = {
  *
  * @example
  * ```jsx
+ * const ref = useRef<HTMLDialogElement>(null);
+ * <Button onClick={() => ref.current?.showModal()}>Open modal</Button>
+ * <Modal
+ *   ref={ref}
+ *   header={{
+ *     label: "Optional label",
+ *     icon: <FileIcon aria-hidden />,
+ *     heading: "My heading",
+ *   }}
+ * >
+ *   <Modal.Body>
+ *     <BodyLong>Hello world</BodyLong>
+ *   </Modal.Body>
+ *   <Modal.Footer>
+ *     <Button>Save</Button>
+ *     <Button type="button" variant="tertiary" onClick={() => ref.current?.close()}>Close</Button>
+ *   </Modal.Footer>
+ * </Modal>
+ * ```
+ * @example
+ * ```jsx
  * const [open, setOpen] = useState(false);
```

Comment:
```suggestion
 * @example
 * Statechange with `useState`
 * ```jsx
 * const [open, setOpen] = useState(false);
```